sky/packages/sky/lib/src/animation/timeline.dart

```diff
@@ -34,24 +106,41 @@ class _TweenSimulation extends Simulation {
   bool isDone(double timeInSeconds) => timeInSeconds > _durationInSeconds;
 }
 
+typedef TimelineCallback(double value);
+
 /// A timeline for an animation
```

Comment:
This comment doesn't tell me what this class does. Specifically, both "timeline" and "animation" are far too vague.